indy_node/test/txn_author_agreement/test_recover_taa_from_ledger.py

```diff
@@ -113,4 +119,16 @@ def test_recover_taa_from_ledger(txnPoolNodeSet,
     assert TXN_AUTHOR_AGREEMENT_RATIFICATION_TS in res_1['result']['data']
     assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_VERSION] == version_1
     assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_RETIREMENT_TS] == retired_time
-    assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_RATIFICATION_TS] == get_txn_time(res_0['result'])
+    assert res_1['result']['data'][TXN_AUTHOR_AGREEMENT_RATIFICATION_TS] == taa_1_ratification_ts
+
+    # Step 14. Disable TAA written using old handler, make sure ratification date is not spoiled
+    disable_res = sdk_send_txn_author_agreement_disable(looper, sdk_pool_handle, sdk_wallet_trustee)[1]
+    res_0 = sdk_get_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet_trustee, version=version_0)[1]
+
+    assert TXN_AUTHOR_AGREEMENT_DIGEST in res_0['result']['data']
```

Comment:
Why don't we check this for the old TAA with `version_1` as well?

Reply:
Added a new check